docs/developer-guide/site.md

```diff
@@ -9,11 +9,12 @@ To test:
 ```bash
 make serve-docs
 ```
+Once running, you can view your locally built documentation at [http://0.0.0.0:8000/](http://0.0.0.0:8000/). 
 
 Check for broken external links:
 
 ```bash
-make lint-docs
+make build-docs
 ```
```

Comment:
I think we can just drop this whole section. `build-docs` doesn't actually check for broken links. And there's really no need to run `build-docs` if everything looks okay after `serve-docs`.